requirements.in

```diff
@@ -0,0 +1,28 @@
+django>=1.11,<1.11.99
+
+git+https://github.com/CalPolyResDev/django-jfu#egg=django-jfu
+git+https://github.com/TK-IT/tktitler.git@v1.1.0#egg=tktitler
+wiki==0.4.5
+
+django-constance[database]
+
+# We can't use newer versions of Pillow, because of django-verstatileimagefield.
+# Since django-verstatileimagefield==1.10 it has pillow 6.0.0 compability.
+Pillow>=2.4.0,<=6.0.0
```

Comment:
Dette kan fjernes, da `pip-compile` er smart nok til at resolve dependencies ordentligt.
Uden denne requirement for man:
```
pillow==6.0.0             # via django-versatileimagefield, wiki
```